resources/gnomad_liftover.py

```diff
@@ -0,0 +1,160 @@
+    """
+ 
+    # filter to snps
+    ht = ht.filter(hl.is_snp(ht.alleles[0], ht.alleles[1]))
+    snp_count = ht.count()
+    logger.info('{} SNPs in table'.format(snp_count))
+
+    # check if reference allele matches what is in reference fasta
+    # for snps on negative strand, make sure reverse complement of ref allele matches what is in reference fasta
+    ht = ht.annotate(
+        reference_mismatch = hl.cond(
+                                ht.new_locus.is_negative_strand,
+                                (reverse_complement(ht.alleles[0]) != hl.get_sequence(ht.locus.contig, ht.locus.position, reference_genome=rg)),
+                                (ht.alleles[0] != hl.get_sequence(ht.locus.contig, ht.locus.position, reference_genome=rg))
+                                )
+                )
+
+    #ht = checkpoint(ht, dtype)
+    return ht
+ 
+def check_mismatch(ht: hl.Table) -> dict:
+    """
+    Checks for mismatches between reference allele and allele in reference fasta
+    :param Table ht: Table to be checked
+    :return: Dictionary containing mismatch type ('total' or 'minus' as key) and umber of mismatches as value
+    :rtype: dict
+    """
+
+    mismatch = {}
+    ht = ht.filter(ht.reference_mismatch)
+    mismatch['total'] = ht.count()
+    mismatch['minus'] =  ht.filter(ht.new_locus.is_negative_strand).count()
+    return mismatch
+
+
+def main(args):
+
+    hl.init()
+    
+    # get reference genomes and load liftover chain file
+    rg37 = hl.get_reference('GRCh37')
+    rg38 = rg38 = hl.get_reference('GRCh38')
+    rg37.add_liftover('gs://hail-common/references/grch37_to_grch38.over.chain.gz', rg38)
+
+    # add fasta sequence to rg38
+    rg38.add_sequence(
+            'gs://hail-common/references/Homo_sapiens_assembly38.fasta.gz', 
+            'gs://hail-common/references/Homo_sapiens_assembly38.fasta.fai'
+            )
+    
+    logger.info('Reading in release ht')
+    if args.type == 'exome':
+        dtype = 'exome'
+        ht = hl.read_table(public_exomes_ht_path(split=True, version=CURRENT_RELEASE))
+    else:
+        dtype = 'genome'
+        ht = hl.read_table(public_genomes_ht_path(split=True, version=CURRENT_RELEASE))
+
+    logger.info('Working on gnomAD {} release ht'.format(dtype))
+    #logger.info('Filtering to chr21 for testing')
+    #ht = ht.filter(ht.locus.contig == '21')
+
+    logger.info('Lifting ht to b38')
+    ht = lift_ht(ht, dtype, rg38)
+
+    logger.info('Checking SNPs for reference mismatches')
+    ht = snp_mismatch(ht, dtype, rg38)
+    mismatch = check_mismatch(ht)
+    logger.info('{} reference mismatches in SNPs'.format(mismatch['total']))
+    logger.info('{} mismatches on minus strand'.format(mismatch['minus']))
+
+
+if __name__ == '__main__':
+
+    # Create argument parser
+    parser = argparse.ArgumentParser(description='This script lifts gnomad ht to b38')
+    parser.add_argument('-t', '--type', help='Data type', choices=['exome', 'genome'], required=True)
```

Comment:
As above, it's up to you if you want to keep it this way, but take a look at how we do the `--exomes` or `--genomes` flags in some of the other scripts (e.g. `generate_frequency_data.py`).